packages/blade/package.json

```diff
@@ -283,14 +283,14 @@
       "name": "Import Button only",
       "path": "./build/components/index.web.js",
       "import": "{ Button }",
-      "limit": "17.5 kb",
+      "limit": "19.5 kb",
       "running": false
     },
     {
       "name": "Import Text only",
       "path": "./build/components/index.web.js",
       "import": "{ Text }",
-      "limit": "13 kb",
+      "limit": "14.5 kb",
```

Comment:
@anuraghazra I have increased limits here since the validate tree shaking job was failing. [CI](https://github.com/razorpay/blade/actions/runs/5313644080/jobs/9636524037)

Reply:
Sounds good.